web/password_prompt.js

```diff
@@ -33,12 +33,12 @@ import { PasswordResponses } from './pdfjs';
 /**
  * @class
```

Comment:
Nice find! Good to know that we can remove these comments as well. Done in the new commit.